airflow/contrib/operators/dataproc_operator.py

```diff
@@ -715,6 +726,9 @@ def __init__(
         self.variables = variables
 
     def execute(self, context):
+        """
```

Comment:
This is really necessary. You should provide a backward compatibility mechanism and add a note in UPDATING.md

Reply:
Could you explain more?

Reply:
Does each operator need a description for the execute method? I do not like it, because the description for this method is already in BaseOperator.execute.

Reply:
Ok, I got it. You are right.